Student/Student/Assignments/AssignmentDetails/AssignmentDetailsPresenter.swift

```diff
@@ -87,14 +93,22 @@ class AssignmentDetailsPresenter {
             return .staged
         }
     }
+    var pageViewLogger: PageViewEventViewControllerLoggingProtocol
 
-    init(env: AppEnvironment = .shared, view: AssignmentDetailsViewProtocol, courseID: String, assignmentID: String, fragment: String? = nil) {
+    init(env: AppEnvironment = .shared,
+         view: AssignmentDetailsViewProtocol,
+         courseID: String,
+         assignmentID: String,
+         fragment: String? = nil,
+         pageViewLogger: PageViewEventViewControllerLoggingProtocol = PresenterPageViewLogger()
```

Comment:
I don't love having one more thing to pass into each of these presenters. Could the logger be on the AppEnvironment?

Reply:
this is a good suggestion.  I like this better as well.  Fixed.